roles/openshift_logging/templates/elasticsearch.yml.j2

```diff
@@ -28,11 +28,10 @@ cloud:
 discovery:
   type: kubernetes
   zen.ping.multicast.enabled: false
-  zen.minimum_master_nodes: {{es_min_masters}}
+  zen.minimum_master_nodes: {{es_node_quorum}}
```

Comment:
They are originally ENV VARS because of the deployer and exist now because of legacy.  We should ask the question regarding all these ENV vars and if they should remain or become an ansible variable.  I assume the threshold is whether or not we desire to tweak the value after deployment.  I assume node quorum and min masters we would prefer to peg those based on ansible inventory where timeouts would be nice to be able to change and tweak?  @ewolinetz ?